src/core/api.proto

```diff
@@ -269,7 +367,9 @@ message InferResponseHeader
     //@@       If specified deliver results for this output as raw tensor data.
     //@@       The actual output data is delivered in the HTTP body for an HTTP
     //@@       request, or in the :cpp:var:`InferResponse` message for a gRPC
-    //@@       request. Only one of 'raw' and 'batch_classes' may be specified.
+    //@@       request. Only one of 'raw' and 'batch_classes' may be
```

Comment:
Raw still needs to be used if results are delivered via shared memory since it delivers meta-data associated with those results. This comment and the one below don't need to change.